docs/concepts/pdp/cloud-pdp-capabilities.mdx

```diff
@@ -0,0 +1,145 @@
+---
+sidebar_position: 2
+title: Cloud PDP Capabilities
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+## What is the Cloud PDP?
+
+The **Cloud PDP** is Permit's **managed Policy Decision Point** that runs in Permit.io's infrastructure and serves authorization decisions over HTTPS.
+It exposes the same high‑level APIs as the container PDP and is **backward compatible** with existing SDK integrations that target
+`https://cloudpdp.api.permit.io`.
+
+Use the Cloud PDP when you want:
+
+- **Managed infrastructure** – No containers, upgrades, or scaling to maintain.
+- **High throughput & data volume** – Optimized for large‑scale policy checks.
+
+For full control over deployment topology, network placement, and container‑level tuning, use the
+[`permitio/pdp-v2` container PDP](/concepts/pdp/overview#run-a-local-pdp-with-docker).
+
+## Supported Policy Models
+
+- **RBAC (Role‑Based Access Control)** – **Supported**
+- **ReBAC (Relationship‑Based Access Control)** – **Supported**
+- **ABAC (Attribute‑Based Access Control)** – **Not supported**
```

Comment:
it's silly to list ABAC under supported just to not it's not. it should probably be in a separate list along with Pbac